Sources/CodeEditSourceEditor/Controller/TextViewController+LoadView.swift

```diff
@@ -29,15 +28,18 @@ extension TextViewController {
             for: .horizontal
         )
 
+        minimapView = MinimapView(textView: textView, theme: theme)
+        minimapView.postsFrameChangedNotifications = true
+        minimapView.isHidden = !showMinimap
+        scrollView.addFloatingSubview(minimapView, for: .vertical)
+
```

Comment:
Should we follow the convention of `setUpHighlighter()` and `setUpTextFormation()` and break this into a `setUpMinimap()` function? We might do this same with the findPanel with `setUpFindPanel()`. Idk if it is that cut and dry though.

Reply:
That's not a bad idea, I thought it might be overkill since it'll be only a few options but I'll move it for consistency.